src/styles/breakpoints.js

```diff
@@ -0,0 +1,22 @@
+const size = {
+  base: 0,
+  mobile: "320px",
+  desktopTablet: "600px",
+  desktopL: "900px",
+  desktopXL: "1280px",
+};
+
+export const minDevice = {
+  base: `(min-width: ${size.base})`,
+  mobile: `(min-width: ${size.mobile})`,
```

Comment:
Shouldn't the min be 0? 🤔 
```suggestion
  mobile: `(min-width: ${size.base})`,
```

Reply:
How about others, should they also be set as "one down", meaning `(min-width: ${size.mobile})` for tablet and so on? 🤔

Reply:
I misunderstood you before. Actually, I would change it one more time :) In few articles i found that this a good practice - https://jsramblings.com/how-to-use-media-queries-with-styled-components/ But if you think differently, I will be happy if I get to know new solutions :)

Reply:
I see, I think I need to see it in action to have an opinion, given my zero experience with styled components ;) thanks for the article, it makes sense to keep it this way. :)